_posts/2020-05-13-easy-steps-to-techdebt.md

```diff
@@ -0,0 +1,160 @@
+it is safe to assume that future whistle blowers think twice before speaking their mind.
+This is a great method for establishing dominance over your team,
+as well as keeping snitches at bay.
+
+## Step 4) Permanent Hurry a.k.a. Eternal Death March
+
+Since we already created the sense of urgency,
+we are now able to ride that wave all the way to the never-ending grind.
+
+The key for this to have minimum of 5 extra tickets added to each sprint,
+to ensure that the developers will have enough work at all times.
+It is a well-known fact,
+that when all the tickets on the sprint are completed,
+developers will cease working completely.
+In order to avoid this standstill,
+we will always have to make sure that there is an absolute minimum of at least 5 extra tickets on the sprint.
+Naturally more the merrier is the rule of thump here.
+It is pretty much a grounded fact there cannot be too many tickets on a single sprint.
+
+![hurry](/img/tech-debt/hurry.jpg)
+
+Remember: If all the tickets on sprint were completed before the midnight preceeding the sprint review/demos,
+the project manager has failed as a supervisor.
+If the tickets are NOT completed by the same deadline,
+the developers have been slacking and should face serious consequences for doing so.
+
+By pushing the developers to their mental and physiological capacity,
+we will ensure that all the corners that can be cut will be cut.
+By running on tighest possible schedule,
+there will be no time for refactorings, ad hoc improvements,
+or god for bid planning of any kind.
+This will lay the ground work for all of the technical debt that you wish to accumulate.
+
+## Step 5) All the work has to be ticketed
+
+No refactoring without tickets. Period.
+
+Under no circumstances should it be allowed for developers to do any sort of ad hoc refactoring.
+If you hear your team using slurs like "See it, fix it",
+you might be already in dealing with this undesired behavior.
+
+It should be made crystal clear for the whole organization,
+that working on anything that is not ticketed, approved, and appointed by the project management,
+is strictly forbidden.
+It should be clarified doing so is a form of time theft,
+and thus will be dealt with as such.
+
+You don't allow watching Netflix at work,
+so why would you allow any other form of time thievery.
+Right?
+
+## Step 6) Avoid tooling
+
+Many programmers are inherently lazy.
+Thus they are always working towards spending company time, money, and resources into reducing their own workloads.
+They call this non-sense "Tooling".
+
+This so called tooling comes in many forms and shapes.
+Starting from code formating tools all the way to expensive dedicated build servers, pipelines, and other massive money drains.
+What these instruments have in common is their purpose.
+To standardize and ease the daily developer work and code deployments.
```

Comment:
Maybe this could be changed from " To standardize and ease the daily developer work and code deployments. " to something around "Make developers look like they are doing work while they make a computer do it"? CI is, after all, just a way for those lazy developers to do anything on their work time and claim it as useful, as per XKCD no. 303 https://xkcd.com/303/